cloudify_nsx/library/nsx_esg_dlr.py

```diff
@@ -995,15 +1071,233 @@ def update_common_edges(client_session, resource_id, kwargs, esg_restriction):
             "Can't change dhcp rules"
         )
 
-    _, routing = common.get_properties_and_validate('routing', kwargs)
+    if esg_restriction:
```

Comment:
At a glance these look the same again (though they're a bit longer than the other two so I might've missed a difference?)